local-repo-committers.py

```diff
@@ -21,31 +21,35 @@ def read_repo_committers(repo_obj):
     """
     repo_authors = {}
     earliest_commit = None
-    try:
-        for commit in repo_obj.iter_commits():
-            if earliest_commit is None:
-                earliest_commit = commit.committed_datetime - timedelta(days_back)
-
-            if commit.committed_datetime > earliest_commit:
-                author = commit.committer.email
-                # skip automation users that look like root@1976d98b6ec0
-                if re.match(r"^root@\w+$", author):
-                    continue
-                if author not in repo_authors:
-                    repo_authors[author] = commit.committed_datetime.strftime("%Y-%m-%dT%H:%M:%S")
+    #try:
+    for commit in repo_obj.iter_commits(repo_obj.refs):
```

Comment:
This is an important change - it now will iterate through all branches, not just the active one. This was potentially suppressing some results.